CaImAn/caiman_run.py

```diff
@@ -186,17 +186,24 @@
     attrs = load_attributes(fname, h5path+'/ImagesStack')
 
     # Parameters
+    params = params_doric.copy()
+    caiman_operation_name = params_doric["Operations"]
     if "OperationName" in params_source_data:
```

Comment:
What if there are both OperationName and Operations?

Reply:
indeed I have to update that